raiden/network/proxies/token_network.py

```diff
@@ -1402,11 +1402,18 @@ def _set_total_withdraw(
                 block_identifier=failed_at_blockhash,
             )
 
-            if detail.state != ChannelState.OPENED:
+            if detail.state > ChannelState.OPENED:
```

Comment:
`ChannelState` is an `IntEnum` so comparisons work.

Reply:
I was never big fan of comparing int values of enums as if you are not careful and add some new value later on in the wrong place it can cause really hard to find bugs. That is just my experience.

Reply:
Ahh, I was looking at https://github.com/raiden-network/raiden/blob/develop/raiden/transfer/state.py#L70, which is a plain `Enum`. Is there a reason for having both or is it just for historic reasons?